frontstage/controllers/party_controller.py

```diff
@@ -251,19 +251,22 @@ def is_respondent_enrolled(party_id, business_party_id, survey_short_name, retur
             return True
 
 
-def change_respondent_status(respondent_id, status):
-    logger.debug('Changing account status', respondent_id=respondent_id, status=status)
+def notify_party_and_respondent_account_locked(respondent_id, email_address, status=None):
```

Comment:
The name before fits the function better, status is passed in so respondent is not always going to be set to 'account_locked' also depending on what status is passed through notify will/will not be called.